src/utils/line.ts

```diff
@@ -189,6 +194,7 @@ export const formatSecuredLineData = (
     positionIds: string[];
     positions: PositionMap;
   };
+  creditEvents: CreditEvent[];
   spigot: { type: CollateralTypes; line: string; tokenRevenue: { [key: string]: string } };
```

Comment:
Why not import spigot and escrow types and use them?

Reply:
fixed